fagiServer/src/test/java/TextMessageIntegrationTests.java

```diff
@@ -78,4 +78,28 @@ void sendingAMessageToConversation_ShouldResultInConversationBeingStored() {
                 .verify(data, times(1))
                 .storeConversation(conversation);
     }
+
+    @Test
+    void callingRun_ShouldCallTickAndSendMesasage() throws InterruptedException {
+        when(data.getConversation(Mockito.anyLong())).thenReturn(conversation);
+
+        inputHandler.handleInput(message);
+
+        var conversationThread = new Thread(conversationHandler);
+        conversationThread.setDaemon(true);
+        conversationThread.start();
+
+        while (conversationHandler.queueSize() > 0 && conversationThread.isAlive()) {
```

Comment:
God idé at sætte:
@Timeout(value = 10, unit = TimeUnit.MINUTES)
Så vi kan terminere hvis det tager for langt tid.